libnethack/include/extern.h

```diff
@@ -533,17 +595,20 @@ extern void display_file(const char *, boolean);
 extern FILE *fopen_datafile(const char *filename, const char *mode, int prefix);
 extern int open_datafile(const char *filename, int flags, int prefix);
 extern char *loadfile(int fd, int *datasize);
-extern int create_bonesfile(char *bonesid, char errbuf[]);
+extern char *bones_filename(const char *bonesid);
+extern int create_bonesfile(const char *bonesid, const char **errbuf);
 extern void commit_bonesfile(char *bonesid);
 extern int open_bonesfile(char *bonesid);
 extern int delete_bonesfile(char *bonesid);
 extern void paniclog(const char *, const char *);
-extern boolean lock_fd(int fd, int retry);
-extern void unlock_fd(int fd);
```

Comment:
I removed `lock_fd` and `unlock_fd` because they were conflicting, and nothing appeared to be using them anymore.